tests/rocker_scripts/matrix.json

```diff
@@ -33,7 +32,7 @@
       "base_image": "rocker/r-ver",
       "tag": "devel",
       "script_name": "install_rstudio.sh",
-      "script_arg": "daily"
+      "script_arg": "latest"
```

Comment:
Seems unrelated change.

Reply:
As you know, daily has been failing for a week due the RStudio server throwing a 500 error on that download.  Because we try to build that image daily in the cron tasks anyway, I think it's less important to also test it here, especially when it means that the current situation ends up blocking the ability to fix anything. Maybe you can provide your preferred solution?

Reply:
Oh, I'm sorry. I didn't know that the daily build was failing. Indeed, as you say, this is used every day, so we don't have to dare to test it here.

I'm sure that `"latest"` was tested elsewhere (because latest is the default), so I think it's okay to delete the test itself.

Reply:
ok thanks.  yeah sorry I get the emails every day when the daily fails (https://github.com/rocker-org/rocker-versioned2/actions/workflows/devel.yml) which happens all the time.  It looked like this test was building rstudio on r-ver:devel while other tests were building it on the normal r-ver so I didn't delete it, but if you're fine dropping this test then so am I!